### ISSUE TITLE
Increase the number of lane changes of surrounding vehicles

### ISSUE THREAD

**Issue by MRMarlies** [Farama-Foundation/HighwayEnv#621]
Hello!

Currently I am working on my Phd and one of my tasks is to train an agent with PPO to learn an adaptive cruise control for an automated vehicle. Therefore, I decided to use the highway-env. 
One thing that blocks my task at the moment is, that I am not able to increase the number of lane changes of the surrounding vehicles. It seems to me as soon as the vehicles reached there target lane, they will never change the lane again. For the training it is quite important that the agent sees also lots vehicles that change the lane to interact with its environment correctly. 
Does anyone have a hint for me how I could adapt the behavior of the vehicles and increase the number of lane changes?

I would be more than happy for every hint!

KR,
Marlies

**Comment by yourczw** [comment]
Yes, I would also like to know this. If your issue is resolved, can you share it with me，thankyou
